### PR TITLE
Multi Table

### PR DESCRIPTION
**Pull request by shaqayq**
In this part of the project, I have created two other tables `Owners `and `Spices`, then join them with the `animals `table.

**What animals belong to Melody Pond?**
melody

**List of all animals that are pokemon (their type is Pokemon).**

**List all owners and their animals, remember to include those that don't own any animal.**

**How many animals are there per species?**

**ist all Digimon owned by Jennifer Orwell.**

**List all animals owned by Dean Winchester that haven't tried to escape.**

**Who owns the most animals?**

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -16,4 +17,16 @@ VALUES
 ('Angemon','2005-06-12',1,true,-45),
 ('Boarmon','2005-06-07',7,true,20.4),
 ('Blossom','1998-10-13',3,true,17),
-('Ditto','2022-05-14',4,true,22);
\ No newline at end of file
+('Ditto','2022-05-14',4,true,22);
+
+
```
Comment (KDlamini) [inline data.sql]: You are missing the following SQL for inserting data into the `owners` and `species` table according to the project requirements.

image

- Kindly make sure that the SQL for all the listed data is present in your `data.sql` file.

File: data.sql [hunk 2]
```diff
@@ -16,4 +17,16 @@ VALUES
...
+/************UPDATE FORIEGNKEY species_id VALUES**************/
+UPDATE animals set species_id=2 where name like '%mon'
+UPDATE animals set species_id=1 where name !~~ '%mon'
+
```
Comment (KDlamini) [inline data.sql:41]: You are also missing the SQL for the following queries in your `queries.sql` file as per project requirements.

image

- Kindly make sure that the SQL for the above queries is present in your `queries.sql` file.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @shaqayq 🙋‍♀️,  

Excellent work! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Good PR title description
✔️  The `schema.sql` file has all required SQl

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (shaqayq) [reply data.sql:41]: @KDlamini !
Thank you for reviewing my code. I will change it. :blush:

**Review by smunozmo — APPROVED** [review]
Hi @shaqayq, 👋 

## Highlights 👏 

- Excellent job making the suggested changes by the last reviewer, great! 😎 
- All transactions are done correctly, perfect! ✅ 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**